mozilla_vpn/explores/channel_group_proportions_table.explore.lkml

```diff
@@ -1,2 +1,10 @@
 include: "../views/channel_group_proportions_table.view"
-explore: channel_group_proportions_table {}
+include: "/shared/views/*"
+explore: channel_group_proportions_table {
+
+  join: countries {
+    type: left_outer
+    relationship: one_to_one
```

Comment:
This probably doesn't really matter since you're presumably just getting dimensions from `countries` and not measures, but it seems like this is a many-to-one.
```suggestion
    relationship: many_to_one
```